specs/binding-filesystem.spec/src/main/resources/META-INF/zilla/filesystem.idl

```diff
@@ -34,5 +35,10 @@ scope filesystem
             string16 tag = null;
             int64 timeout = 0;
         }
+
+        struct FileSystemResetEx extends core::stream::Extension
+        {
+            string16 error;
```

Comment:
This should be specific error codes (via enum) rather than freeform strings, so that other bindings can know what to expect in terms of different possible errors.